Content.Server/Psionics/Invisibility/PsionicInvisibilitySystem.cs

```diff
@@ -36,10 +36,16 @@ private void OnInit(EntityUid uid, PotentialPsionicComponent component, Componen
             SetCanSeePsionicInvisiblity(uid, false);
         }
 
+        /// <summary>
+        /// Being able to see invisible by default is no longer tracked by "Not having Potential Psionic".
+        /// Anything intended to be immune to invisibility(and mind magic in general) should instead have PsionicInsulation as a built-in component
```

Comment:
Make sure this is listed in the PR. Not sure if my laptop can handle reloading 124 files so just gonna leave this comment here.